bentoml/_internal/cli/model_management.py

```diff
@@ -172,37 +176,47 @@ def delete_target(target):
 
                 if delete_confirmed:
                     model_store.delete(model.tag)
-                    click.echo(f"{model} deleted")
+                    logger.info(f"{model} deleted")
 
         for target in delete_targets:
             delete_target(target)
 
-    @model_cli.command(help="Export Model to a tar file")
+    @model_cli.command()
     @click.argument("model_tag", type=click.STRING)
-    @click.argument(
-        "out_file", type=click.File("wb"), default=sys.stdout, required=False
-    )
-    def export(model_tag, out_file):
-        """Export Model files to a tar file
+    @click.argument("out_path", type=click.STRING, default="", required=False)
+    def export(model_tag, out_path):
+        """Export Model files to an archive file
+
+        arguments:
+
+        \b
+        MODEL_TAG: model identifier
+        OUT_PATH: output path of exported model.
+          If this argument is not provided, model is exported to name-version.bentomodel in the current directory.
+          Supported formats are tar('tar'), tar.gz ('gz'), tar.xz ('xz'), tar.bz2 ('bz2'), and zip
 
+        examples:
+
+        \b
+        bentoml models export FraudDetector:latest > my_model.tar
```

Comment:
does this syntax still work ? now we've changed the click argument type from `click.File` to `click.String`?  it is ok to not support this syntax, but let's make sure the documentation is correct